packages/reactivity/src/constants.ts

```diff
@@ -22,6 +22,10 @@ export enum ReactiveFlags {
   RAW = '__v_raw',
 }
 
+export enum RefFlags {
+  IS_REF = '__v_isRef',
```

Comment:
Can we just move it to `ReactiveFlags`?

Reply:
I have also considered this. However, ReactiveFlags seems to be specifically designed for the `reactive` API. If we want to move it over, you should check whether the name ReactiveFlags needs to be changed.

Reply:
yeah, when I started writing a reactivity summary, this can indeed be moved to `ReactiveFlags`.

Reply:
done